scripts/travis_install.sh

```diff
@@ -7,4 +7,10 @@ hash -r
 conda config --set always_yes yes --set changeps1 no
 conda update -q conda
 conda info -a
-conda install jupyter
+
+# install development version of juptyer
+# https://github.com/jupyter/notebook/blob/master/.travis.yml
+git clone https://github.com/jupyter/notebook notebook-dev
+cd notebook-dev
+git clone --quiet --depth 1 https://github.com/minrk/travis-wheels travis-wheels
+pip install -f travis-wheels/wheelhouse --pre file://$PWD#egg=notebook[test] coveralls
```

Comment:
`--pre` shouldn't be needed here. Further, since we aren't running the notebook tests, this should be able to be:

    pip install -f travis-wheels/wheelhouse .